apps/betterangels-admin/vite.config.mts

```diff
@@ -0,0 +1,58 @@
+/// <reference types='vitest' />
+import { nxCopyAssetsPlugin } from '@nx/vite/plugins/nx-copy-assets.plugin';
+import { nxViteTsPaths } from '@nx/vite/plugins/nx-tsconfig-paths.plugin';
+import react from '@vitejs/plugin-react';
+import path from 'path';
+import { ProxyOptions, defineConfig } from 'vite';
+
+const SERVER_PORT = 8083;
```

Comment:
should this be the same as shelter? same question for the preview port on line 41